config/packages/oneup_flysystem.yaml

```diff
@@ -0,0 +1,17 @@
+# Read the documentation: https://github.com/1up-lab/OneupFlysystemBundle/tree/master/Resources/doc/index.md
+oneup_flysystem:
+    adapters:
+        default_adapter:
```

Comment:
what is it for?
It doesn't seem to be used.

Reply:
it's the default installed with the bundle. I didn't mind to remove it. thanks for the heads up. removed now.